runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/util/ThreadLocalByteStringOutputStream.java

```diff
@@ -89,11 +89,12 @@ private static StreamHandle getStreamHandleFromThreadLocal() {
     // softRefHolder is only set by Threadlocal initializer and should not be null
     SoftRefHolder softRefHolder =
         Preconditions.checkArgumentNotNull(threadLocalSoftRefHolder.get());
-    StreamHandle streamHandle;
+    StreamHandle streamHandle = null;
```

Comment:
can you add Nullable annotation to this and softReference?

Reply:
Done.